react/features/chat/middleware.ts

```diff
@@ -573,21 +576,33 @@ function _handleReceivedMessage({ dispatch, getState }: IStore,
 
     // Provide a default for the case when a message is being
     // backfilled for a participant that has left the conference.
-    const participant = getParticipantById(state, participantId) || { local: undefined };
+    const participant = getParticipantById(state, participantId);
```

Comment:
Is this removed by accident? If no, why? Also we should probably remove the comment as well.